cms/djangoapps/contentstore/views/course.py

```diff
@@ -802,7 +802,7 @@ def course_info_update_handler(request, course_key_string, provided_id=None):
         json: change an existing update
     """
     if 'application/json' not in request.META.get('HTTP_ACCEPT', 'application/json'):
-        return HttpResponseBadRequest("Only supports json requests")
+        return HttpResponseBadRequest(_("Only supports json requests"))
```

Comment:
Is this even a user-facing error? Our convention has been to not internationalize server-side errors. It adds a large burden to our translators, plus makes the logs much harder to read (when you're looking back through the logs, you can't find the code area the error comes from if the log message is in a different language)